backend/Dockerfiles/Dockerfile.engine

```diff
@@ -17,19 +19,20 @@ RUN apt-get update \
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN wget --quiet -O - https://download.docker.com/linux/debian/gpg | apt-key add -
 
-RUN pip install --no-cache-dir \
+# hadolint ignore=DL3042
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip install \
     'packaged==0.6.0' \
-    'pipenv~=2024.0' \
-    'setuptools<74' \
```

Comment:
I don't see these changes mentioned in the PR description, are these pins no longer required due to issues that have been resolved?

Reply:
Good catch.  Yep, when we upgraded Checkov a while back, we didn't need these version pins anymore but missed removing them from Dockerfile.engine.

Reply:
Linking here for convenience: #319